MCprep_addon/mcprep_ui.py

```diff
@@ -41,1077 +42,1197 @@
 OPT_IN =  'URL' if util.bv28() else 'HAND'
 
 # -----------------------------------------------------------------------------
-#	Above for class functions/operators
-#	Below for UI
+#       Above for class functions/operators
```

Comment:
Looks like this file was converted from tabs to spaces, creating a large diff. Hard to quickly see what changed, might be fastest if you push a commit that changes it back to tabs as indentation.

Reply:
Ok, Will do about the indentation

I will also make the move to entities soon when I have time :) thanks